src/main.c

```diff
@@ -62,6 +63,7 @@ int debugflag = 0;
 int editor = EDITOR_JOE;
 char rttyoutput[120];
 int use_vk = 0;
+int use_qtc_vk = 0;
```

Comment:
Does that variable make sense? It is needed no where (just as the variable above).